.gitignore

```diff
@@ -35,3 +35,7 @@
 /config/master.key
 
 coverage/*
+
+coverage/*
```

Comment:
Probably leftover after rebase :) One `coverage/*` is enough.

Reply:
Bravo me :(. Check :)